docs/additional-configuration.adoc

```diff
@@ -215,6 +215,16 @@ kubectl create secret -n "$NAMESPACE" generic git-ssh-key \
   --from-file=ssh_config=/tmp/ssh_config \
   --from-literal=passphrase="$PASSPHRASE"
 ----
++
+Note: If a passphrase is provided, the DevWorkspace operator adds a postStart event that starts the SSH agent and adds the passphrase.
+The DevWorkspace operator also modifies the bashrc file to configure the `SSH_AGENT_PID` and `SSH_AUTH_SOCK` environment variables.
```

Comment:
- "operator" should also be capitalized here for consistency
- Maybe we should replace all mentions of "bashrc" with `` `~/.bashrc` ``?

Reply:
Fixed both